client/src/locales/fr-FR.json

```diff
@@ -1513,6 +1513,13 @@
                 "status": "Status",
                 "download": "Télécharger"
             },
+            "status": {
+                "paid": "Payée",
+                "ready": "À payer",
+                "draft": "Brouillon",
+                "uncollectible": "Non trouvée",
+                "void": "Inconnu"
```

Comment:
`uncollectible` doesn't mean `not found`, it means that we can't enforce the payment. `void` doesn't mean `unknown`, it means that it's not legally binding.

You can find an explanation for every status on the Stripe website https://docs.stripe.com/invoicing/overview.